multisense_ros/cfg/multisense.cfg

```diff
@@ -197,6 +197,9 @@ for cfg in sensorConfigList:
         gen.add("lighting", bool_t, 0, "Lighting", False)
         gen.add("flash", bool_t, 0, "Flash", False)
         gen.add("led_duty_cycle", double_t, 0, "DutyCycle", 0.0, 0.0, 1.0)
+        if cfg.ar0234:
+            gen.add("number_of_pulses", int_t, 0, "NumberOfPulses", 1, 0, 10)
+            gen.add("led_startup_time", int_t, 0, "LedStartupTime", 0, -10000, 10000)
```

Comment:
Can we add a unit of time in the `led_startup_time` name?

Reply:
I updated to `led_startup_time_us`, and `LedStartupTimeUs`